CHANGELOG.md

```diff
@@ -104,6 +104,7 @@ No changes to highlight.
 * Make try examples button more prominent by [@aliabd](https://github.com/aliabd) in [PR 2705](https://github.com/gradio-app/gradio/pull/2705)
 * Fix id clashes in docs by [@aliabd](https://github.com/aliabd) in [PR 2713](https://github.com/gradio-app/gradio/pull/2713)
 * Fix typos in guide docs by [@andridns](https://github.com/andridns) in [PR 2722](https://github.com/gradio-app/gradio/pull/2722)
+* Add pt-br.json by [@pstwh](http://github.com/pstwh) in [PR 2753](https://github.com/gradio-app/gradio/pull/2753)
```

Comment:
This should be higher up in the changelog, in the "Upcoming Release" section, not in the "3.12.0" section

Reply:
Fixed, sorry for my inattention.

Reply:
No worries, thanks for the fix!